wagtail/test/testapp/models.py

```diff
@@ -1083,6 +1108,34 @@ def get_preview_template(self, request, mode_name):
 register_snippet(PreviewableModel)
 
 
+class CustomPreviewSizesModel(PreviewableMixin, models.Model):
+    text = models.TextField()
+
+    @property
+    def preview_sizes(self):
+        return [
```

Comment:
It's the same config for `CustomPreviewSizesPage` so I guess we can just define a list at the top and use it for both